src/main/java/com/surrealdb/refactor/driver/SurrealDBWebsocketClientProtocolHandler.java

```diff
@@ -24,42 +27,41 @@ public class SurrealDBWebsocketClientProtocolHandler
     private static final Logger log =
             Logger.getLogger(SurrealDBWebsocketClientProtocolHandler.class.toString());
     private static final String PROPERTY_REQUEST_ID = "id";
+    private final ConcurrentMap<String, Promise<JsonObject>> requestMap = new ConcurrentHashMap<>();
     private ChannelPromise handshakeFuture;
-    private final ConcurrentMap<String, Promise<JsonObject>> requestMap = new ConcurrentHashMap();
-
     private Channel channel;
 
     public SurrealDBWebsocketClientProtocolHandler() {}
 
     @Override
-    public void handlerAdded(ChannelHandlerContext ctx) {
-        handshakeFuture = ctx.newPromise();
+    public void handlerAdded(final ChannelHandlerContext ctx) {
+        this.handshakeFuture = ctx.newPromise();
```

Comment:
Same comment about `this` - we don't want to keep typing it, so please revert

Reply:
Removed all the `.this`.